var/spack/repos/builtin/packages/totalview/package.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
We still need the ending copyright year to be current.

```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Reply:
@tldahlgren 
Nice catch.  I'm probably still working with the original package from when I started looking at this last year.  Hence the old copyright and nicholas sly as one of the maintainers.   I also just found a problem with trying to install TotalView on a Mac with this version.  I get a message about there being no checksum.  Isn't that what the sha256 strings are?   And then the install fails if I chose to install anyway   I could remove the Mac lines for this pull request if that is acceptable.